### ISSUE TITLE
HEARTBEAT logs

### ISSUE THREAD

**Issue by satrik** [dgreif/ring#1065]
# Bug Report

### Describe the Bug
 Since I've updated to v11.3.0, the plugin generates a lot of these logs (see below)

### To Reproduce
 - No steps required

### Expected behavior
 Don't log these messages.

### Screenshots/Logs
 

```
HEARTBEAT sending PING { last_stream_id_received: 5 }
HEARTBEAT PONG { lastStreamIdReceived: 3 }
HEARTBEAT sending PING { last_stream_id_received: 6 }
HEARTBEAT PONG { lastStreamIdReceived: 4 }
```

### Additional context
None.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```
        {
            "refreshToken": "xxx",
            "debug": false,
            "_bridge": {
                "username": "xxx",
                "port": 54318
            },
            "platform": "Ring"
        },
```

### Environment

 - OS: Raspberry Pi OS/Debian GNU/Linux 11 (bullseye)
 - Node.js: v16.17.0
 - NPM: 8.15.0
 - homebridge-ring: v11.3.0
 - homebridge: v1.5.0

**Comment by solarbeat** [comment]
FWIW, downgraded to v11.2.1 and I'm still getting these heartbeat messages.

**Comment by JacVid** [comment]
Hello, 

I have exactly the same issues as Satrik and Solarbeat. Isn't it related to FFMPEG ? 

FFMPEG has been updated with v11.3.0, so even if we downgrade to v11.2.1, we still have the issue...

Any how, getting so much log is really annoying. So please, if a solution is available, let us know.

Thanks.

**Comment by tsightler** [comment]
But literally no one in this thread has said what problem they are having.  These heartbeat messages are not likely to be the problem as they are just log messages from the push-receiver.  The likely "fix" is just to hide the messages from the log, but that would not impact functonality.

Can someone in this thread please state what actual problem they are experiencing vs just assuming that the log messages are related to whatever that problem is?

**Comment by tsightler** [comment]
Well, @solarbeat did say that it "seems to be affecting functionality" and @JacVid spoke of FFMPEG, which has nothing to do with this message, so I assume they were attempting to associate these extra log messages with some other issue they were having.  If the only complaint is that the messages exist well, yeah, that's fair.

**Comment by solarbeat** [comment]
I did note that it may be coincidence, as before I downgraded, nothing Ring related seemed to be working at all within Homebridge, but without any error messages, that could've been anything. The restart I did after downgrading to 11.2.1 might've been the kick necessary to get things working again. Again, nothing in the logs except for the heartbeat messages, just adding possibly spurious/anecdotal data points.

**Comment by Sungamm** [comment]
Good Morning,
I have the same issue. With issue I mean my log is full of this entries

`HEARTBEAT sending PING { last_stream_id_received: 127 }
HEARTBEAT PONG { lastStreamIdReceived: 125 }
HEARTBEAT sending PING { last_stream_id_received: 128 }
HEARTBEAT PONG { lastStreamIdReceived: 126 }
HEARTBEAT sending PING { last_stream_id_received: 129 }
HEARTBEAT PONG { lastStreamIdReceived: 127 }
HEARTBEAT sending PING { last_stream_id_received: 130 }
HEARTBEAT PONG { lastStreamIdReceived: 128 }`

But functionallity is working fine. 

Nevertheless its hard to read the log and do to the fact there is no timestemp in the log its complete useless for me. Is there any option I can do from user level?

Or is an update of the ring-plugin the only solution? I understand from discussion before that a roll back is not helpful?

**Comment by satrik** [comment]
@Sungamm either you pin a previous version (3.0.2) of push-receiver in package.json and re install the plugin or you change the the client.js inside  `node_modules/@eneris/push-receiver` and just remove all `console.log` lines

**Comment by JacVid** [comment]
> @Sungamm either you pin a previous version (3.0.2) of push-receiver in package.json and re install the plugin or you change the the client.js inside `node_modules/@eneris/push-receiver` and just remove all `console.log` lines

How do I get to this directory via the Homebridge Terminal ? I'm a newbie on linux, sorry, and the Homebridge Terminal is my only way to interact with Homebridge and OS...

Thanks for your help

**Comment by satrik** [comment]
@JacVid the path depends on your OS, but under Linux it should mostly be `/usr/lib/node_modules/homebridge-ring/node_modules/@eneris/push-receiver/dist`. To edit client.js you need to be root or use sudo with vi/vim/nano or any other editor

**Comment by JacVid** [comment]
I'm using Raspbian GNU/Linux Bullseye (11), and in usr/lib, I can't find node_modules directory :(

**Comment by satrik** [comment]
`npm list -g` should list all installed modules and the path

**Comment by JacVid** [comment]
Thanks a lot !

I could reach /usr/local/lib/node_modules/homebridge-ring/node_modules/@eneris/push-receiver/dist

As said I'm a newbie in Linux, so how do you edit client.js file via Homebridge Terminal ?

**Comment by satrik** [comment]
Just Google how to use nano on the terminal and don't forget to run nano with sudo.

**Comment by JacVid** [comment]
@satrik thanks a lot.

I could use Nano to edit the file and remove the console.log lines... First, my Homebridge session still works for now with this modification... Second my Ring doorbell seems to work correctly.

Now let's see if the HEARTBEAT log occurs again ;)

Once again, thank you for your help.

**Comment by Sungamm** [comment]
> my [PR](https://github.com/Eneris/push-receiver/pull/3) got merged and Eneris released a new version (3.1.2) of push-receiver. reinstalling the plugin or running `npm update` in the plugins directory will upgrade the package and the unneeded logging should stop.
> 
> thanks @tsightler for the push-receiver hint

Work perfect! Thank you for the prompt and friendly help!
